.github/workflows/fuzzers.yml

```diff
@@ -96,6 +96,15 @@ jobs:
           - afl_virginmap
           - afl_maxmap
           - introspector_driven_focus
+          - libfuzzer_focus_idx0
```

Comment:
meh, don't bother putting all of these in our CI. This is too much

Reply:
Right. I wonder if this will fail for future people adding fuzzers. I guess if it does, then it is not worth it.

Reply:
Yes! it may cause others see their CI failing too.

Reply:
Should I skip registering it and ignore presubmit failure?

Reply:
Nah, leave as is please.